addons/place-backpack-code-at-cursor/userscript.js

```diff
@@ -0,0 +1,45 @@
+export default async function ({ addon }) {
+  const Blockly = await addon.tab.traps.getBlockly();
+  const {
+    traps: { vm },
+    redux,
+  } = await addon.tab;
```

Comment:
This syntax seems confusing but that could just be me.

Reply:
It was this nice one liner before then the auto formatter ruined it. Either way, whilst it is maybe a little more confusing it has the benefits of only using one await, and less duplicate code. I'd say it doesn't matter.